packages/storybook/stories/components/button/button.stories.tsx

```diff
@@ -164,3 +174,136 @@ export const AccessibilityLabelledBy: StoryObj = {
     </>
   ),
 }
+
+export const AccessibilityBadPracticeLabelledBy: StoryObj = {
+  decorators: [(story) => <div style={{ display: 'flex', flexFlow: 'row', gap: '8px', alignItems: 'center' }}>{ story() }</div>],
+  tags: ['!dev'],
+  render: ({}) => (
+    <>
+      <Button>
+        <Icon name={ICON_NAME.filter} />
+      </Button>
+      <span>Filter your search results</span>
+    </>
+  ),
+}
+
+export const AccessibilityBadPracticesRoleStatus: Story = {
+  parameters: {
+    docs: {
+      source: { ...staticSourceRenderConfig() },
+    },
+  },
+  render: () => {
+    const [message, setMessage] = useState('');
+
+    const handleClick = () => {
+      setMessage('Copied to clipboard.');
+    };
+
+    return (
+      <>
+        <Button onClick={ handleClick }>
+          Copy
+        </Button>
+
+        <span
+          style={{ marginLeft: '1rem' }}
+        >
+          { message }
+        </span>
+      </>
+    );
+  },
+};
+
+export const AccessibilityRoleStatus: Story = {
+  parameters: {
```

Comment:
`tags: ['!dev']` otherwise the story is added to the list of button stories on the sidebar